src/filter/filters.ts

```diff
@@ -166,6 +166,32 @@ export const handleAbiDecode = (context: any, filter: { $abi: Abi }) => {
   }
 }
 
+export const handleAbstractAbiDecode = (context: any, filter: any) => {
```

Comment:
can `filter` and `context` be typed? I think we should avoid using `any` in a function signature

Reply:
at a minimum it looks like we can add `$abiAbstract` to filter